salt/states/archive.py

```diff
@@ -1148,6 +1162,15 @@ def extracted(name,
                         )
                     )
                     return ret
+                if __opts__['test'] and clean_parent and contents is not None:
+                    ret['result'] = None
+                    ret['comment'] += (
+                        ' Since the \'clean_parent\' option is enabled, the '
+                        'destination parent directory would be removed first '
+                        'and than re-created and the archive would be '
```

Comment:
Typo:
```suggestion
                        'and then re-created and the archive would be '
```

Could also be re-written to include the actual path that will be purged (using `.format`). But definitely fix the typo.

Reply:
thanks, fixed